src/main/java/com/twilio/Twilio.java

```diff
@@ -150,6 +154,17 @@ public static synchronized void setEdge(final String edge) {
         Twilio.edge = edge;
     }
 
+    /**
+     * Set the logger configuration file path.
+     *
+     * @param path path to logging configuration file
+     */
+    public static synchronized void setLoggerConfiguration(final String path) {
```

Comment:
nit: can we make the input variable name a smidge more specific. Maybe `filePath`?